johnzon-core/src/main/java/org/apache/johnzon/core/JsonStreamParserImpl.java

```diff
@@ -34,6 +34,8 @@
 
 //This class represents either the Json tokenizer and the Json parser.
 public class JsonStreamParserImpl extends JohnzonJsonParserImpl implements JsonChars {
+    private static final Event[] EVENT_VALUES = Event.values();
```

Comment:
No need to define this here again, in `org.apache.johnzon.core.JsonCahrs` there's already a `Event[] EVT_MAP =Event.values()`

Reply:
Fixed.